notebooks/Image_Analysis.ipynb

```diff
@@ -326,6 +382,51 @@
     "plt.legend(loc='best')\n",
     "plt.show()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64253971",
+   "metadata": {},
+   "source": [
+    "### Line Plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5d8eb201",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hist_a = plt.hist(horizontal_coeff_a, bins=np.logspace(np.log10(min(horizontal_coeff_a)), np.log10(max(horizontal_coeff_a))), label='PSP-A', alpha=0.5)\n",
+    "hist_b = plt.hist(horizontal_coeff_b, bins=np.logspace(np.log10(min(horizontal_coeff_b)), np.log10(max(horizontal_coeff_b))), label='PSP-B', alpha=0.5)\n"
```

Comment:
```min(horizontal_coeff_a)``` might some time return 0 and raise an error in ```np.log10()```. In my experiments, I do ```np.log10(min(horizontal_coeff_a)+0.1)```. Either adding 0.1 or any other method to avoid the error should be good.

Reply:
Great catch! It should actually be `np.log10( max( [ min(horizontal_coeff_a), np.finfo(float).eps ] ) )`.

Reply:
The full expression is:

`bins = np.logspace(min, max)`

The `np.logspace` starts the range at `1` with `np.finfo(float).eps` as `min` of the range.

Reply:
@Saurav-D is the notebook up to date with this solution?

Reply:
Yes.